onboarding/src/Components/hooks/Packages.js

```diff
@@ -49,20 +49,10 @@ export const fetchPackagesAndForms = (count) => {
       }
   };
 
-  useEffect(() => {
-      fetch(url + "api/package_pages", fetchProps)
-      .catch(error => {
-          showError(error.message);
-          console.error(error);
-      })
-      .then(res => res.json())
-      .then((result) => {
-        setPackagesAndPages(result.sort((a, b) => b.id - a.id) || []);
-      })
-      .finally(() => setIsLoading(false));
-  }, [count]);
+  const { data:packagesAndForms, isLoading, error } = useFetch(`${url}api/package_pages`, fetchProps, count);
+  packagesAndForms && packagesAndForms.sort((a, b) => b.id - a.id) || [];
```

Comment:
`ESLint: Expected an assignment or function call and instead saw an expression.(no-unused-expressions)`
Check for potential logic bug
or just use `// eslint-disable-next-line react-hooks/rules-of-hooks`